amr-acr-mapping.json

```diff
@@ -4,13 +4,15 @@
     "PIN" :  [{ "type": "PIN" }],
     "OTP" :  [{ "type": "OTP" }],
     "Wallet" :  [{ "type": "WLA" }],
-    "L1-bio-device" :  [{ "type": "BIO", "count": 1 }]
+    "L1-bio-device" :  [{ "type": "BIO", "count": 1 }],
+    "Knowledge" : [{"type":"Knowledge"}]
```

Comment:
Can we keep this as "Knowledge" : [{"type":"KBA"}]. I will change the jira tasks accordingly